cli/command/container/attach.go

```diff
@@ -140,7 +143,18 @@ func runAttach(dockerCli command.Cli, opts *attachOptions) error {
 	if errAttach != nil {
 		return errAttach
 	}
-	return getExitStatus(ctx, dockerCli.Client(), opts.container)
+
+	// get status code
```

Comment:
This seems like a perfect place to create a new function. A function is always better than an inline comment as it provides a scope for the documentation.

I think the unit test you removed could be preserved and used to test this, with a few small changes to the test setup. The 3 cases in the table test are still relevant.

Reply:
It feels weird given that it's really just a select, the comment could be removed without affecting readability, but I'll try that.